test/WasmSpec/testsuite/imports.wast

```diff
@@ -177,6 +177,7 @@
   (import "spectest" "global" (global $x i32))
   (global $y (import "spectest" "global") i32)
 
+  (import "spectest" "global" (global i64))
```

Comment:
You should remove that line.
Imported Int64 global is not legal in javascript, thus this cannot pass with our host